ldap/servers/slapd/util.c

```diff
@@ -1067,6 +1067,16 @@ slapi_rand_r(unsigned int *randx)
     return (int)(*randx & 0x7FFFFFFF);
 }
 
+int
+slapi_rand_r_64(uint64_t *randx)
```

Comment:
Does this change belong to that PR ?

Reply:
Ok, so for Y2K38 time_t is going to become 64 bit (versus 32 bit).  So a lot of these changes are to address the upcoming Y2K38 issues.  So we need a 64 bit version of slapi_rand_r() for passwd_extop.c

Reply:
I do not think we really need to change that code as the time is only used to get a random seed value.
  That is the rare case were we could safely truncated the 64 bit time_t to a 32 bit unsigned integer   (upper bits are not really significant as they rarely change anyway) 
Using a 32 bit seed seems good enough especially as we want to generate a  small  random value ...